File: DentiSmart.API/DentiSmart.Domain/Models/Usuario.cs
using MongoDB.Bson;
using MongoDB.Bson.Serialization.Attributes;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Globalization;
using System.Text;

namespace DentiSmart.Domain.Models
{
    [BsonIgnoreExtraElements]
    public class Usuario
    {
        [BsonId]
        [BsonRepresentation(BsonType.ObjectId)]
        public string Id { get; set; }
        public string Nombre { get; set; }
        public string ApellidoPat { get; set; }
        public string ApellidoMat { get; set; }
        public string NombreUsuario { get; set; }
        public string Contrasenia { get; set; }
        public string Email { get; set; }
        public string Role { get; set; }
        public string Telefono { get; set; }
        #nullable enable
        public string? Foto { get; set; }
        public Expediente? Expediente { get; set; }

        [BsonRepresentation(BsonType.ObjectId)]
        public string? Consultorio {get;set;}
        public Consultorio? DatosConsultorio { get; set; }
        public string? HoraInicio { get; set; }
        public string? HoraFin { get; set; }
        public string? IdFacebook { get; set; }
        public string? IdGoogle { get; set; }
    }
}

File: DentiSmart.App/app/src/main/java/com/dreamteam/dentismart/models/Usuario.kt
package com.dreamteam.dentismart.models

import java.io.Serializable

data class Usuario(
        val apellidoMat: String,
        val apellidoPat: String,
        val consultorio: String,
        val contrasenia: String,
        val datosConsultorio: Any,
        val email: String,
        val foto: String,
        val id: String,
        val nombre: String,
        val nombreUsuario: String,
        val horaFin: Any,
        val horaInicio: Any,
        val role: String,
        val telefono: String
):Serializable{
    override fun toString(): String = ("$nombre $apellidoPat $apellidoMat")
}